purger/src/rubin/nublado/purger/purger.py

```diff
@@ -7,11 +7,10 @@
 import datetime
 from pathlib import Path
 
-import structlog
 import yaml
-from safir.logging import Profile, configure_logging
-from safir.slack.blockkit import SlackTextBlock
+from safir.logging import configure_logging
 from safir.slack.webhook import SlackRouteErrorHandler
+from structlog.stdlib import BoundLogger, get_logger
```

Comment:
Importing these tings from `structlog.stdlib` instead of `structlog` gives us type-safe versions of them... though not type-safe enough to catch this original bug ☹️